contexts/AuthContext.tsx

```diff
@@ -199,13 +207,17 @@ export const AuthProvider = ({ children, publicPage = false }: any) => {
 
     return await zencode_exec(keypairoomClient, { data: zenData }).then(({ result }) => {
       const res = JSON.parse(result);
-      setItem("eddsa_public_key", res.eddsa_public_key);
-      setItem("eddsa_key", res.keyring.eddsa);
-      setItem("ethereum_address", res.keyring.ethereum);
-      setItem("reflow", res.keyring.reflow);
-      setItem("schnorr", res.keyring.schnorr);
-      setItem("eddsa", res.keyring.eddsa);
+      setItem("eddsaPrivateKey", res.keyring.eddsa);
+      setItem("ethereumPrivateKey", res.keyring.ethereum);
+      setItem("reflowPrivateKey", res.keyring.reflow);
+      setItem("bitcoinPrivateKey", res.keyring.bitcoin);
+      setItem("ecdhPrivateKey", res.keyring.ecdh);
       setItem("seed", res.seed);
+      setItem("ecdhPublicKey", res.eddsa_public_key);
```

Comment:
you are assigning the `eddsa_public_key` to the `ecdh_public_key`

Reply:
💯